lib/pack/goal-state/cancelGoals.ts

```diff
@@ -204,11 +203,11 @@ async function cancelGoalSet(goalSetId: string, ctx: HandlerContext, id?: string
 
     if (result && result.SdmGoalSet && result.SdmGoalSet.length === 1) {
         const gs = result.SdmGoalSet[0];
-        const newGoalSet = {
+        const newGoalSet: any = {
```

Comment:
Is there not a type for goal set?

Reply:
Those graphql query types and the SdmGoalSetMessage aren’t compatible. The query types are all optional. 😞